homeassistant/helpers/translation.py

```diff
@@ -43,28 +49,35 @@ def component_translation_path(
     """Return the translation json file location for a component.
 
     For component:
-     - components/hue/.translations/nl.json
+     - components/hue/translations/nl.json
 
     For platform:
-     - components/hue/.translations/light.nl.json
+     - components/hue/translations/light.nl.json
 
     If component is just a single file, will return None.
     """
     parts = component.split(".")
     domain = parts[-1]
     is_platform = len(parts) == 2
 
-    if is_platform:
-        filename = f"{parts[0]}.{language}.json"
-        return str(integration.file_path / ".translations" / filename)
-
     # If it's a component that is just one file, we don't support translations
     # Example custom_components/my_component.py
     if integration.file_path.name != domain:
```

Comment:
I think that `is_platform` is now always filtered out by this check?

Reply:
Took a look at the tests (which passed) and there are checks there for this.